app/models/nic/bmc.rb

```diff
@@ -12,7 +12,7 @@ def virtual
     end
     alias_method :virtual?, :virtual
 
-    register_to_enc_transformation :type, ->(type) { type.constantize.humanized_name }
+    attr_exportable :provider, :username, :password => ->(nic) { nic.decrypt_field(nic.password) }
```

Comment:
makes sense, previously we decrypted all encrypted values but being explicit is probably better